RevenueCat/Plugins/Android/PurchasesWrapper.java

```diff
@@ -596,6 +602,52 @@ public void onError(ErrorContainer errorContainer) {
         });
     }
 
+    public static void getEligibleWinBackOffersForProduct(String productIdentifier) {
+        // NOOP
+        PurchasesError error = new PurchasesError(
+                PurchasesErrorCode.UnsupportedError,
+                "Win-back offers are not supported on Android.");
+
+        ErrorContainer errorContainer = PurchasesErrorKt.map(
+                error,
+                new HashMap<>());
+        sendError(errorContainer, GET_ELIGIBLE_WIN_BACK_OFFERS_FOR_PRODUCT);
```

Comment:
I guess we could have moved some of this logic to PHC like we do for getPromotionalOffer:
```
ErrorContainer errorContainer = CommonKt.getPromotionalOffer();
        sendError(errorContainer, GET_PROMOTIONAL_OFFER);
```
But probably not worth changing it right now 👍